server/src/visitors/YmlVisitorHelper.ts

```diff
@@ -0,0 +1,62 @@
+import { FieldContext } from '../grammar';
+import { connection } from '../server';
+
+const BEGINNING_QUOTES_REGEX = /^("""|")\s*/;
+const ENDING_QUOTES_REGEX = /\s*("""|")$/;
+
+export function getDocumentation(fieldOptions: FieldContext[]): string {
+    try {
+        for (const element of fieldOptions.filter((elem) => !!elem.commonField)) {
+            const option = element.commonField();
+            if (
+                !option ||
+                !option._optionName ||
+                option._optionName.text !== 'documentation' ||
+                !option._optionValue ||
+                !option._optionValue.text
+            ) {
+                // There is no option, or option name isn't “documentation” or there is no text value associated.
+                continue;
+            }
+            let _documentation = option._optionValue.text;
+            _documentation = _documentation.replace(BEGINNING_QUOTES_REGEX, '');
+            _documentation = _documentation.replace(ENDING_QUOTES_REGEX, '');
+            return _documentation;
+        }
+    } catch (err) {
+        if (!!err) {
+            connection.console.error(`${err}`);
+        } else {
+            connection.console.error(`An unexpected error occured when getting documentation value.`);
+        }
+    }
+    return null;
+}
+
+export function getType(fieldOptions: FieldContext[], baseType?: string): string {
+    let domains = baseType ? baseType : 'Object';
```

Comment:
https://en.wikipedia.org/wiki/Elvis_operator

> TypeScript supports this concept with its nullish-coalescing operator ??, e.g. (a ?? b), since v3.7.[16]

LOL “nullish-coalescing” xD
Not sure if it is the same as falsy stuff, though. The empty strings seem to be kept, at least in the JS proposal https://github.com/tc39/proposal-nullish-coalescing. Grmbl.